ADOPTERS.md

```diff
@@ -8,6 +8,7 @@
 * [Matrix](https://matrix.org/)
 * [Northwestern Mutual](https://www.northwesternmutual.com/)
 * [Nets](https://www.nets.eu/)
+* [PalFish](https://ipalfish.com/klian/web/dist/teacher/home.html)
```

Comment:
If you prefer, you can also provide two links: the Chinese version of your website, and the English page that you listed here.

Reply:
Thanks for the advice, I've done this with the latest commit